elliott/elliottlib/cli/find_bugs_golang_cli.py

```diff
@@ -317,6 +317,8 @@ def is_valid(b: JIRABug):
             # Do not touch embargoed bugs
             if b.bug.fields.security.name == "Embargoed Security Issue":
                 return False
+            if self.bug.fields.issuetype.name == "Vulnerability" and getattr(self.bug.fields, JIRABugTracker.field_cve_is_embargo).value == "True":
+                return False
```

Comment:
It would be nice to move this into a method like `bug.is_embargoed()`

Reply:
I was trying to check if there such is_embargoed() function but can't find, I am not sure our embargo manage process, right now I only find here, we can abstract to the function if we see other use places

Reply:
seems we don't need to change here because if new CVE bug is embargoed he security level will also be set to embargo level